src/scripts/release.sh

```diff
@@ -23,20 +23,20 @@ fi
 
 ORIGIN_URL=$(git remote get-url origin)
 UPSTREAM="anz-bank"
-UPSTREAM_URL=$(echo "$ORIGIN_URL" | sed -n "s/\\(.*\\/\\/github.com\\/\\)\\(..*\\)\\(\\/..*.git\\)/\\1$UPSTREAM\\3/p")
+UPSTREAM_URL=$(echo "$ORIGIN_URL" | sed -n "s|\\(.*//github.com/\\)\\(..*\\)\\(/..*.git\\)|\\1$UPSTREAM\\3|p")
 
 echo "------- Checkout master ---------"
 git checkout master || fatal "Cannot checkout master"
 
 echo "------- Pull upstream ---------"
 git pull "$UPSTREAM_URL" master || fatal "Cannot pull  upstream master"
 
-if [[ $COMANND = "prepare" ]]; then
+if [[ $COMMAND = "prepare" ]]; then
     RELEASE_BRANCH="release-v$VERSION"
-    ORIGIN=$(echo "$ORIGIN_URL" | sed -n "s/.*\\/\\/github.com\\/\\(..*\\)\\/..*.git/\\1/p")
-    REPO=$(echo "$ORIGIN_URL" | sed -n "s/.*\\/\\/github.com\\/..*\\/\\(..*\\).git/\\1/p")
+    ORIGIN=$(echo "$ORIGIN_URL" | sed -n "s|.*//github.com/\\(..*\\)/..*.git|\\1|p")
+    REPO=$(echo "$ORIGIN_URL" | sed -n "s|.*//github.com/..*/\\(..*\\).git|\\1|p")
```

Comment:
Some comment here re EREs, single quotes and `sed -n`

Reply:
Done.